src/foam/util/Emails/EmailsUtility.java

```diff
@@ -100,6 +100,8 @@ public static void sendEmailFromTemplate(X x, User user, EmailMessage emailMessa
       return;
     }
 
+    emailMessage.setBody( emailMessage.getBody().replaceAll("\\.svg", ".png") );
```

Comment:
Feeling it should be before email service being called so before this cts.apply(userX, group, emailMessage, templateArgs); above.